### ISSUE TITLE
Receiving reports that REDIS can write, read .. but cannot clear the cache (via /cache/index)

### ISSUE THREAD

**Issue by seansan** [colinmollenhour/Cm_Cache_Backend_Redis#135]
I am getting repports that REDIS can write, read .. but cannot clear the cache

Cache is filling up
But when we clear cache and go to REDIS managament we see the same number of keys
Also we see outdated data on the frontend (so the cache is not cleared)
And a next indicator is: the cache clearing is just too fast

Did something change in past updates like below? (I read something about READ ONLY ...)

This is our version, is it not supported?

`Redis server v=5.0.1 sha=00000000:0 malloc=jemalloc-5.1.0 bits=64 build=fb0cee1c3dbaca2d`

**Comment by colinmollenhour** [comment]
I don't think I've tried v5 yet but I also don't see any reason it would not work the same as previous versions. There isn't much for me to go on here...

**Comment by seansan** [comment]
I am preparing a downgrade to 3.2 and let you know if this was the problem.

**Comment by seansan** [comment]
Nope. Sorry. There is something wrong and I am reviewing updates in both Redis, Nginx and M1 to find where it was introduced. Are you having similar issues or interested?
